speechbrain/dataio/dataio.py

```diff
@@ -31,7 +33,7 @@
 logger = logging.getLogger(__name__)
 
 
-def load_data_json(json_path, replacements={}):
+def load_data_json(json_path: str, replacements: Dict = {}, target_columns: Union[List, None] = None):
```

Comment:
From what I could see we are lacking unit tests for the `load_data_csv`/`load_data_json`, I feel like those should be added and should test the column filtering functionality.